libraries/EmbeddedChessPieces/src/ECPMovement/ECPMovement.h

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file ECPMovement.h
+ * @author Ines Rohrbach, Nico Schramm
+ * @brief Helper class for chess piece movement.
+ * @version 0.1
+ * @date 2024-11-12
+ * 
+ * @copyright Copyright (c) 2024
+ * 
+ */
+
+#ifndef ECPMovement_h
+#define ECPMovement_h
+
+#include <Dezibot.h>
+#include <EmbeddedChessPieces.h>
+
+#define FORWARD_TIME 750
+#define MOVEMENT_BREAK 375
+
+
+class ECPMovement {
+public:
+    /**
+     * @brief Construct a new movement object.
+     * 
+     * @param d Dezibot to move
+     * @param mc Base value to calibrate the dezibot's movement
```

Comment:
Add here that the default value is 3900.

Maybe we should extract the default value to a constant in there (`#define DEFAULT_MOVEMENT_CALIBRATION`)? If so, we can remove the comment in the doc string of `ECPMovement.movementCalibration`.